profile.md

```diff
@@ -228,6 +228,25 @@ For example:
 ]
 ```
 
+### JSON Schema Restrictions
+
+Platforms MAY enforce restrictions on the JSON schemas service brokers can use
+to describe the input parameters they accept (see [spec](spec.md#schema-object).
+Each section below will describe any restrictions a Platform places on input
+parameter schemas.
+
+#### Cloud Foundry
+
+The following restrictions are enforced within a Cloud Foundry deployment:
```

Comment:
hmm, these are requirements on brokers right? So, are we saying that a broker that wants to be used by both Kube and CF may need to have different schema?  I guess I'm asking whether these CF requirements are really CF specific or if they should be true for all platforms/brokers?

Reply:
I was hoping they wouldn't be clashing requirements - just additive things like fields that should be expressed (rather than "Broker authors must not do ...").